### ISSUE TITLE
Raiding the lower cells to themselves when expanding the table

### ISSUE THREAD

**Issue by ValeriyJefimov** [fuzz-productions/FZAccordionTableView#38]
I want to make the table expandable to its full size with every change. To do this, I increase the altitude constant for each launch of WillOpenSection and reduce it in WillCloseSection. But Lower cells hide themselves while animation. Is there are any way to fix this? Thanx for help.

**Comment by kgaidis** [comment]
Could you show a video of what you are trying to accomplish and what is the issue? I'm not quite getting what's going on.

Right now I'm assuming u are trying to achieve the following? 
![](https://github.com/fuzz-productions/FZAccordionTableView/raw/master/Images/Second_Example.gif)

**Comment by ValeriyJefimov** [comment]
No, I'm trying to do this. I've resolve this issue with not good way - just move all expanding code
 to DIDopen/close section. 
Thanx a lot for answering.

![ezgif com-optimize](https://cloud.githubusercontent.com/assets/19861161/25907292/b78bb7c8-35af-11e7-8633-01532ef60ae1.gif)

**Comment by kgaidis** [comment]
I guess you mean that you "resolved" it by waiting for the opening or closing to complete, and only then opening the next one.

And it seems like the buginess is with the header rather than cells?

It's been a while since I first created the library, but this might be one of those issues where `UITableView` just performs these default animations and we kind of are at the mercy of what it's doing. That's not to say that there isn't a solution, I'm just not 100% sure what it would be without doing a deep dive. 

The general hope is that `FZAccordionTableView` sticks to as much default `UITableView` behavior as possible since it's supposed to be a "light wrapper."

There is a comment I left here where I faded out cells upon show/delete:
https://github.com/fuzz-productions/FZAccordionTableView/issues/7#issuecomment-209948099

There might be a chance that some of this animation buginesss could be mitigated by playing around how cells animate in / out.
